src/cli/onefuzz/templates/libfuzzer.py

```diff
@@ -66,7 +66,6 @@ def _create_tasks(
         colocate_all_tasks: bool = False,
         colocate_secondary_tasks: bool = True,
         check_fuzzer_help: bool = False,
-        no_check_fuzzer_help: bool = False,
```

Comment:
no_check_fuzzer_help is the new one, and that's the one that we set


check_fuzzer_help <- is the one that should be removed, no ?

Reply:
This is the internal `_create_tasks` function. The bug was that the value of no_check_fuzzer_help ignored here. the template function (including q_emu) pass the value of `not no_check_fuzzer_help` to this function